MediaGalleryMetadata/Model/SegmentNames.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryMetadata\Model;
+
+/**
+ * Segment types to names mapper
+ */
+class SegmentNames
+{
+    private const SEGMENT_TYPE_TO_NAME = [
```

Comment:
Do we need to use hexadecimal values here? I guess it makes it harder to understand and when debugging the application.